manifests/application/consul.pp

```diff
@@ -17,7 +17,23 @@
   },
 )
 {
-  include ::consul
+  class { "::consul":
+#    install_method       => 'package',
+    config_defaults      => hiera_hash('consul::config_hash'),
```

Comment:
config_defaults is populated by the consul::config_hash key? Is that intentional?